package.json

```diff
@@ -24,7 +24,7 @@
     "change": "beachball change",
     "check-for-changed-files": "cd scripts && yarn fluentui-scripts check-for-modified-files",
     "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
-    "depcheck": "lage depcheck",
+    "depcheck": "rnx-dep-check && lage depcheck",
```

Comment:
Why do we need. both? Are we not quite wanting to change all the subrepo depchecks to rnx-depcheck?

Reply:
`@rnx-kit/dep-check` is more about managing dependencies and aligning them within and across repos. You can read more about it here: https://github.com/microsoft/rnx-kit/blob/main/packages/dep-check/DESIGN.md

Whereas depcheck's purpose is to find unused or missing dependencies. They don't overlap in functionality.

Reply:
The name clash is a bit unfortunate. If you have better suggestions, I'm all ears.

Reply:
Thanks for that doc. That was very helpful in explaining what the PR is trying to achieve. Definitely a problem I've run into before with this repo =)

`dep-align`? I'm sure you've already thought of that though